docs/project.toml

```
@@ -0,0 +1,3 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Stingray = "2045e982-6adb-4e09-8364-0d2cc8a3fe4f"
```

This file should be called `Project.toml`, not `project.toml` (case is important).  Also, `Stingray` shouldn't be here and it'd be better to specify compat bounds (latest version of `Documenter.jl` at the moment is [`v0.27.18`](https://github.com/JuliaDocs/Documenter.jl/releases/tag/v0.27.18))
```suggestion

[compat]
Documenter = "0.27"
```